setup.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Setup script."""
+
+from __future__ import absolute_import
+
+from setuptools import find_packages
+from setuptools import setup
+
+VERSION = '1.0.0'
```

Comment:
This should probably not start at 1.0.0. A 1.0 version implies full stability (API and otherwise) and this package has not been sufficiently tested in the wild. Let's start at 0.1.0.

Please export VERSION in `__init__` as `__version__` attribute (so that people can do: `import x; x.__version__`

Reply:
Done.